### PR TITLE
[Bug 1268454] [Feature] Display the "display name" instead of the "user name" when possible

### PR DESCRIPTION
**Pull request by MikkCZ**
Need to check, if the problem is only here, or other places too.

### CODE REVIEW

File: kitsune/users/templatetags/jinja_helpers.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def unicode_to_html(text):
 def user_list(users):
     """Turn a list of users into a list of links to their profiles."""
     link = u'<a class="user" href="%s">%s</a>'
-    list = u', '.join([link % (escape(profile_url(u)), escape(u.username)) for
+    list = u', '.join([link % (escape(profile_url(u)), escape(display_name(u))) for
```
Comment (MikkCZ) [inline kitsune/users/templatetags/jinja_helpers.py]: fixes https://bugzilla.mozilla.org/show_bug.cgi?id=1268454#c0

File: kitsune/wiki/jinja2/wiki/includes/revision_list.html [hunk 2]
```diff
@@ -73,7 +73,7 @@
           </div>
           <div class="creator">
             <a href="{{ profile_url(rev.creator) }}">
-              {{ rev.creator }}
+              {{ display_name(rev.creator) }}
```
Comment (MikkCZ) [inline kitsune/wiki/jinja2/wiki/includes/revision_list.html]: Fixes https://bugzilla.mozilla.org/show_bug.cgi?id=1268454#c1

**Comment by MikkCZ** [comment]
I hope I haven't missed any user name in the code, where it shouldn't be, and vice versa.

r?

**Comment by MikkCZ** [comment]
@safwanrahman can you please check, if I haven't put the display name somewhere, where the username is actually intended?

**Comment by safwanrahman** [comment]
@MikkCZ I will check it as soon as possible.

**Comment by glogiotatidis** [comment]
Thanks for the PR. Due to ongoing discussions about the future of the platform we'll avoid working on any pull requests not directly related to security or maintenance. I've marked this pull request as "Platform Decision Pending" to flag it for later review.

**Comment by glogiotatidis** [comment]
Closing PR since development has been halted. Thanks your contribution.

**Pull request closed without merging**